src/frontend/screens/AppPasscode/EnterPasscode.tsx

```diff
@@ -1,16 +1,21 @@
 import * as React from "react";
-import { View } from "react-native";
+import { SecurityContext } from "../../context/SecurityContext";
 import Button from "../../sharedComponents/Button";
 import Text from "../../sharedComponents/Text";
-import { SecurityContext } from "../Security/SecurityContext";
 
 export const EnterPasscode = () => {
-  const { setPassIsSet } = React.useContext(SecurityContext);
+  const [authState, setAuthState] = React.useContext(SecurityContext);
 
   return (
     <React.Fragment>
-      <Text>Under Construction</Text>
-      <Button onPress={() => setPassIsSet(false)}>Turn Off App Passcode</Button>
+      <Text>{authState.passcode}</Text>
+      <Button
+        onPress={() => {
+          setAuthState({ type: "setPasscode", newPasscode: null });
+        }}
+      >
+        Turn Off App Passcode
```

Comment:
Add translation string for this

Reply:
This is just placeholder code! working on changing all this stuff this sprint